File: cpp/funkcje.cpp
/*
 * funkcje.cpp
 * 
 */


#include <iostream>

using namespace std;

void dodaj(int a, int b) // dleklaracja funkcji
{
    cout << "Suma: " << a+b << endl;
}

int odejmij (int l1, int l2)
{
    return l1-l2;
}

void mnoz(int c, int d)
{
    cout << "Iloczyn: " << c*d << endl;
}

int dziel (int e, int f)
{
    if (f == 0)
    {
        cout << "Nie dzieli sie przez 0!" << endl;
        return 0;
    }
        return e/f;
    
}

int main(int argc, char **argv)
{
    int a, b;
    a = b = 0;
    
    cout << "Podaj dwie liczby:" << endl;
    cin >> a >> b;
    
    dodaj(a, b); // wywołanie funkcji 
    cout << "Różnica: " << odejmij (a, b) << endl;
    mnoz (a, b);
    cout << "Iloraz: " << dziel (a, b) << endl;
    
    return 0;
}

File: python/funkcje.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-


def sumuj(a, b):
    return a + b


def odejmij(a, b):
    return a - b


def mnoz(a, b):
    return a * b


def dziel(a, b):
    return a + b


def main(args):
    a = int(input("podaj 1. liczbę: "))
    print(a)
    b = int(input("podaj 2. liczbę: "))
    print(b)

    print("suma: ", sumuj(a, b))
    print("różnica: ", odejmij(a, b))
    print("iloczyn: ", mnoz(a, b))
    print("iloraz: ", dziel(a, b))

    return 0


if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
